assets/src/js/cookies.js

```diff
@@ -10,7 +10,7 @@ export const setupCookies = () => {
   const NECESSARY_ANALYTICAL = '3';
   const NECESSARY_ANALYTICAL_MARKETING = '4';
 
-  const ALL_COOKIES = ENABLE_ANALYTICAL_COOKIES ? NECESSARY_ANALYTICAL_MARKETING : NECESSARY_MARKETING;
```

Comment:
Hi Stefan, Could you please elaborate on why we dropped this condition? If a user accepts all cookies and ENABLE_ANALYTICAL_COOKIES is unset from the admin settings, the Greenpeace cookie was set to 2 (i.e., NECESSARY_MARKETING), but now it's set to 4 (i.e., NECESSARY_ANALYTICAL_MARKETING).

Reply:
Thanks for the detailed info, I got your point 👍